release-notes/rascal-0-40-x-release-notes.md

```diff
@@ -0,0 +1,92 @@
+---
+authors: [jvinju]
+title: "Rascal 0.40.x release notes"
+sidebar_position: 96
+---
+
+In this post we report on the Rascal release 0.40.x
+
+<!--truncate-->
+
+## Release 0.40.14 - November 5, 2024
+
+The public release 0.40.x follows release 0.28.x; many improvements have been made in projects that depends on the rascal interpreter and the standard library (the type checker, the VScode extensions, clair, etc.) Some of these improvements depend directly on fixes in the interpreter and additions to the standard library. 
```

Comment:
We need to have a paragraph that describes the new tpl&checker situation. What users will have to do, and what in the future they don't have to anymore.

Reply:
you're right! That's not this package but it is Rascal in general. I'll have a look also at the maven plugin diff.